docs/interfaces/json-api.md

```
@@ -152,7 +165,7 @@ udpn.recv | bool | Indicates whether broadcast packet reception is enabled. To s
 udpn.sgrp | 0 to 255 | Bitfield for broadcast send groups 1-8
 udpn.rgrp | 0 to 255 | Bitfield for broadcast receive groups 1-8
 udpn.nn | bool | Don't send a broadcast packet (applies to just the current API call). Not included in state response.
-v | bool | If set to _true_ in a JSON POST command, the response will contain the full JSON state object. Not included in state response
+v | bool | If set to _true_ in a JSON POST command, the response will contain the full JSON object of the requested path instead of `{"success":true}`: the state object when posting to `/json/state`, state and info for `/json/si`, and so on. Not included in state response
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)